Detectors/CTP/workflowIO/src/DigitReaderSpec.cxx

```diff
@@ -58,29 +62,79 @@ class DigitReader : public Task
 DigitReader::DigitReader(bool useMC)
 {
   if (useMC) {
-    LOG(info) << "CTP does not support MC truth at the moment";
+    LOG(info) << "CTP : truth = data as CTP inputs are already digital";
   }
 }
 
 void DigitReader::init(InitContext& ic)
 {
   auto filename = o2::utils::Str::concat_string(o2::utils::Str::rectifyDirectory(ic.options().get<std::string>("input-dir")),
                                                 ic.options().get<std::string>("ctp-digit-infile"));
+  LOG(info) << "init: " << ic.options().hasOption("ignore-irframes") << " " << ic.options().get<bool>("ignore-irframes");
```

Comment:
Note that this line breaks the FullCI with `][ERROR] Exception caught while in Init: missing option: ignore-irframes (No such node (ignore-irframes)). Exiting with 1.` since you unconditionally query this option

Reply:
removed